python/observation-publisher/publisher/config.py

```diff
@@ -0,0 +1,160 @@
+from dataclasses import dataclass
+from enum import Enum, unique
+from typing import List, Optional, Dict
+
+import arize.utils.types
+from hydra.core.config_store import ConfigStore
+
+
+@unique
+class ValueType(Enum):
+    FLOAT64 = 1
+    INT64 = 2
+    BOOLEAN = 3
+    STRING = 4
+
+
+@dataclass
+class RegressionPredictionOutput:
+    prediction_score_column: str
+
+    @property
+    def column_types(self) -> Dict[str, ValueType]:
+        return {self.prediction_score_column: ValueType.FLOAT64}
+
+
+@dataclass
+class BinaryClassificationPredictionOutput:
+    prediction_label_column: str
+    prediction_score_column: Optional[str] = None
```

Comment:
in arize the prediction_score is optional, but from model standpoint the prediction score is mandatory and prediction label is optional (can be derived from prediction score). We need to update this later on when introducing model schema

Reply:
Noted, will add a TODO inline comment.